### PR TITLE
Update README.md [INT2-54] Introduction - Spanish #351 - RM3

### PR DESCRIPTION
**Pull request by RestlessMike**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,26 @@
-# Introducción a Yearn
+#Introducción
```
Comment (DarkGhost7) [inline README.md:1]: Keep spacing the same here please dont change the markdown format. 
this needs to be # Introducción

File: README.md [hunk 2]
```diff
@@ -1,40 +1,26 @@
...
 
-Esta documentación está en continuo desarrollo por parte de la comunidad de Yearn.
+Yearn Finance es un grupo de productos financieros dentro de las finanzas descentralizadas o (DeFi) cuyo objetivo es generar rendimiento de capital en plataformas con contratos inteligentes como Ethereum. El protocolo es mantenido por desarrolladores independientes y gobernado por holders del token YFI.
 
-Yearn Finance es un conjunto de productos financieros descentralizados \(DeFi\) centrados en crear una forma sencilla de generar rendimientos de riesgo ajustado para los depositantes de diferentes activos a través de los mejores protocolos de préstamos, pools de liquidez y estrategias de yield farming desarrolladas por la comunidad en Ethereum.
+Puedes encontrar una breve descripción de los productos centrales de Yearn, el proceso de gobernanza y los enlaces para los canales de comunicación activos al pie de esta página.
 
-## Productos principales
+##Productos centrales
+###Vaults
```
Comment (DarkGhost7) [inline README.md:8]: there is a space here between lines 7 and 8 in the raw english version

File: README.md [hunk 3]
```diff
@@ -1,40 +1,26 @@
...
 
-### Vaults
+Fondos de liquidez que generan rendimiento de capital basados en oportunidades presentes en el mercado. Las Vaults benefician a los usuarios socializando el coste del gas, automatizando la generación de rendimiento de capital y balanceando el proceso así como moviendo el capital a medida que surgen nuevas oportunidades. El usuario final no necesita tener conocimientos de los protocolos subyacentes involucrados en Defi por lo que las vaults representan una estrategia de inversión pasiva.
 
-- Robots de yield farming desarrollados por la comunidad.
-- Las yVaults siguen una estrategia única que está diseñada para maximizar el rendimiento del activo depositado y minimizar el riesgo.
+###Earn
```
Comment (DarkGhost7) [inline README.md:12]: space between 12 and 13

File: README.md [hunk 4]
```diff
@@ -1,40 +1,26 @@
...
-### Earn
+##Gobernanza 
+El ecosistema de Yearn está controlado por los holders del token YFI los cuales se someten y votan en las propuestas off-chain que gobiernan el ecosistema. Las propuestas que generan apoyo mayoritario (>50% de los votos) son implementados por la cartera multifirma conformada por 9 miembros. Los cambios deben ser firmados por al menos 6 de los 9 miembros de la cartera multifirma para poder ser implementados. Los miembros de la cartera multifirma han sido elegidos por votación por los holders de YFI y son sujetos a cambio a través de futuros votos de gobernanza. Por favor refiérase a nuestras FAQ para la [lista de miembros de cartera multifirma](https://docs.yearn.finance/resources/faq#who-is-on-the-multisig) y más información sobre [nuestro proceso de gobernanza](https://docs.yearn.finance/resources/faq#governance).  
 
-- Money markets dinámicos que maximizan el rendimiento y que son usados como bloques de construcción para yVaults.
-- Earn mueve sus fondos de forma automática entre distintos proveedores de préstamos como dydx, Aave o Compound.
+##Canales de comunicación
 
-### Zap
+[Governance Forum](https://gov.yearn.finance/)
```
Comment (DarkGhost7) [inline README.md:20]:
```suggestion
[Governance Forum](https://gov.yearn.finance/)
```
maybe translate Governance Forum

**Review by DarkGhost7 — CHANGES_REQUESTED** [review]
Please fix the issues raised follow the formatting seen here https://raw.githubusercontent.com/yearn/yearn-docs/master/README.md

**Review by DarkGhost7 — APPROVED** [review]
Looks good thanks

**Comment by miguel567** [comment]
Tiny comment: @RestlessMike  Please, remove the parenthesis from DEFI in this phrease `Yearn Finance es un grupo de productos financieros dentro de las finanzas descentralizadas o (DeFi)`

**Pull request closed without merging**